MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -609,6 +619,10 @@
      and is used to enable or disable playback of DRM controlled content. -->
     <string name="security_options_drm_content">Play DRM-Controlled Content (&lt;a href=&quot;http://somesite.com/&quot;&gt;Learn More&lt;/a&gt;)</string>
 
+    <!-- This string labels an Allow/Don't Allow switch in the Privacy and Security settings dialog
+     and is used to enable or disable playback of DRM controlled content. -->
+    <string name="security_options_drm_content_v1">Automatically allow sites to identify your device to play DRM-controlled content. (&lt;a href=&quot;%1$s&quot;&gt;Learn More&lt;/a&gt;)</string>
```

Comment:
Same question I've asked elsewhere: is the HTML supposed to look escaped like this? Is there any way to avoid it?

Reply:
We have been doing this for a while whenever we need to inline links. Unfortunately we need to encode certain symbols inside the xml.

Reply:
Using a CDATA section is a better way to encode these. Our toolchain understands those, and displays them as the inside of the CDATA, and serializes them back like that.